PokeDemo/Controllers/Base/BaseViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  BaseViewController.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+// MARK: - Navigation
+
+import UIKit
+
+class BaseViewController: UIViewController, ReuseIdentifying {
+    
+    // MARK: - Exposed Methods
+    
+    func moveToViewController(storyboard: String, destination: String) -> UIViewController {
+        let storyboard = UIStoryboard(name: storyboard, bundle: nil)
+        let vc = storyboard.instantiateViewController(withIdentifier: destination)
+        return vc
+    }
+}
```

Comment:
Is it used anywhere?

Reply:
Covered in cf0322761f261fbbe3e6c7997b28c2d417663ac9